docs/errors/usage_errors.md

```diff
@@ -728,13 +728,35 @@ try:
         a: int = 1
 
         @field_validator('a')
-        def check_a(self, values):
-            return values
+        def check_a(self, value):
+            return value
 
 except PydanticUserError as exc_info:
     assert exc_info.code == 'validator-instance-method'
 ```
 
+## `json_schema_input_type` used with the wrong mode {#validator-input-type}
+
+This error is raised when you explicitly specify a value for the `json_schema_input_type`
+argument and `mode` isn't set to either `'before'`, `'plain'` or `'wrap'`.
```

Comment:
Let's add one sentence to the docs here explaining why you can't use `json_schema_input_type` with an `after` validator, conceptually.